src/components/pages/pricing/plans/data/plans.json

```diff
@@ -51,19 +51,19 @@
       "rows": "2",
       "feature": {
         "title": "Compute (monthly)",
-        "subtitle": "Compute-hours = CPU size x active time"
+        "subtitle": "Compute hours = CU x active hours"
```

Comment:
Should we use "CU" or "compute size" in the formula? Compute size is easier to understand. You don't have to know what CU means.

Reply:
I agree Compute size is easier to understand.